src/org/labkey/test/components/ui/EntityBulkInsertDialog.java

```diff
@@ -88,6 +91,43 @@ public String getFieldWithId(String id)
         return getWrapper().getFormElement(Locator.tagWithId("input", id));
     }
 
+    public EntityBulkInsertDialog setDateField(String fieldKey, String dateString)
```

Comment:
I'm still not a fan of fieldKey as a parameter name because it is not an attribute in the tag, id and name are.
But this is a discussion better had in the Test Team Working Session meeting.

Reply:
FWIW, I like the term here because it is also literally the column name, in addition to being an attribute of the field row, and the ID of the inputs that aren't reactSelects.  For them, we're kinda stuck using label text